doc/classes/VisualServer.xml

```diff
@@ -1457,6 +1462,7 @@
 			<argument index="0" name="probe" type="RID">
 			</argument>
 			<description>
+				Returns the energy multiplier for this GI Probe. Equivalent to [member GIProbe.energy].
```

Comment:
I previous descriptions you capitalized as "GI probe", but here (and after) "GI Probe". I'd tend to use "GI probe" everywhere, unless it's an actual reference to the GIProbe node.